packages/portalnetwork/src/networks/state/manager.ts

```diff
@@ -0,0 +1,99 @@
+import { Trie } from '@ethereumjs/trie'
+import { Account } from '@ethereumjs/util'
+
+import { ContentLookup } from '../contentLookup.js'
+
+import { PortalTrieDB } from './portalTrie.js'
+import { StateNetworkContentKey } from './util.js'
+
+import type { StateNetwork } from './state.js'
+import type { Path } from '@ethereumjs/trie'
+
+export class StateManager {
```

Comment:
Is this going to implement the EthJS Statemanager interface ultimately?

Reply:
No.  This will remain a readonly state interface, meant to retrieve state via local and network content.  This is the non-bridge client version of a state network client.  a non-bridge node has no need for creating new state, it just needs the methods to walk tries using the network.